src/extensions/try-convert/MSBuild.Abstractions/ProjectExtensions.cs

```diff
@@ -50,8 +50,8 @@ public static string GetTargetFramework(this IProject project)
                 ".NETCore" => "net", // UWP
                 ".NETCoreApp" => "netcoreapp",
                 ".NETPortable" => "netstandard",
-                "MonoAndroid" => "net",
-                "Xamarin.iOS" => "net",
+                "MonoAndroid" => "net7.0-android",
+                "Xamarin.iOS" => "net7.0-ios",
```

Comment:
I suspect that we don't want to force net7.0 here. Is there a way to get what version of .net core that the user wants to update to? e.g. we probably want to use "6.0", "7.0" or "8.0" (now that net8.0 is out) depending on what the user wants to target.

That said, on line 66 someone has already assumed "net6.0-windows" for Windows.

Reply:
Everything for MAUI is hardcoded to 7.0 in the CLI, which is fine until 8.0 ships.